deploy/stacks/container.py

```diff
@@ -44,11 +47,20 @@ def __init__(
         self._ecr_repository = ecr_repository
         self._vpc = vpc
         self._prod_sizing = prod_sizing
+        self._email_custom_domain = email_custom_domain
+        self._ses_configuration_set = ses_configuration_set
```

Comment:
The object variables `self._email_custom_domain` and `self._ses_configuration_set` are not used anywhere, we can remove them

Reply:
Done!